Nos/Views/RepliesView.swift

```diff
@@ -18,6 +19,7 @@ struct RepliesView: View {
     @State private var reply = EditableNoteText()
     
     @State private var alert: AlertState<Never>?
+    @State var showReplyComposer = false
```

Comment:
This should be `private` so we maintain the same style across all SwiftUI views.